sources/packages/backend/apps/api/src/route-controllers/dynamic-form/models/form.dto.ts

```diff
@@ -1,6 +1,23 @@
-import { MaxLength } from "class-validator";
+import { JSON_200KB } from "../../../constants";
+import { JsonMaxSize } from "../../../utilities/class-validation";
+import { IsObject, MaxLength } from "class-validator";
 
 export class FormNameParamAPIInDTO {
   @MaxLength(200)
   formName: string;
 }
+
+export class FormUpdateAPIInDTO {
+  @IsObject()
+  @JsonMaxSize(JSON_200KB)
+  formDefinition: unknown;
+}
+
+export class FormAPIOutDTO {
+  title: string;
+  name: string;
```

Comment:
I would say that we must retrieve the path and use the path unless I misunderstood on the expectation of PUT using the path instead of form name.

Reply:
I replaced it with the path now. We have been using formName as "the path" since always.